monty/exts/core/eval.py

```diff
@@ -1,14 +1,17 @@
 """Evaluation features for Monty. These commands provide the ability to evaluate code within the bot context."""
 
+from __future__ import annotations
```

Comment:
Would like to avoid __future__ annotations in any files in monty/exts, other than that this looks good.